app/src/utils/domain/per.ts

```diff
@@ -44,3 +44,55 @@ export function getFormattedComponentName(component: PerComponent): string {
 
     return `${prefix} : ${title}`;
 }
+
+export const perRatingColors = [
+    'var(--go-ui-color-dark-blue-40)',
+    'var(--go-ui-color-dark-blue-30)',
+    'var(--go-ui-color-dark-blue-20)',
+    'var(--go-ui-color-dark-blue-10)',
+    'var(--go-ui-color-gray-40)',
+    'var(--go-ui-color-gray-30)',
+];
+
+export const perRatingColorMap: {
+    [key: string]: string;
+} = {
+    5: 'var(--go-ui-color-dark-blue-40)',
+    4: 'var(--go-ui-color-dark-blue-30)',
+    3: 'var(--go-ui-color-dark-blue-20)',
+    2: 'var(--go-ui-color-dark-blue-10)',
+    1: 'var(--go-ui-color-gray-40)',
+    0: 'var(--go-ui-color-gray-30)',
+};
+
+export const perAreaColorMap: { [key: number]: string } = {
+    1: 'var(--go-ui-color-purple-per)',
+    2: 'var(--go-ui-color-orange-per)',
+    3: 'var(--go-ui-color-blue-per)',
+    4: 'var(--go-ui-color-teal-per)',
+    5: 'var(--go-ui-color-red-per)',
+};
+
+export const perBenchmarkColorMap: {
+    [key: string]: string;
+} = {
+    1: 'var(--go-ui-color-dark-blue-40)',
+    2: 'var(--go-ui-color-dark-blue-30)',
+    5: 'var(--go-ui-color-dark-blue-10)',
+};
```

Comment:
Why do we have missing numbers for this map?

Reply:
The ID keys are from the backend.

Reply:
Let's add a NOTE. What does these color map with? What's 1, 2 and 5?